.github/workflows/dispatch.yml

```diff
@@ -1,18 +0,0 @@
-name: Trigger repository dispatch event
-on:
-  push:
-    branches:
-      - master
-      - dev
-jobs:
-  dispatch:
-    name: Repository Dispatch
-    runs-on: ubuntu-latest
-    if: github.repository_owner == 'WEC-Sim'
-    steps:
-    - uses: peter-evans/repository-dispatch@v1
-      with:
-        token: ${{ secrets.REPO_ACCESS_TOKEN }}
-        repository: WEC-Sim/WEC-Sim_Applications
-        event-type: wecsim-${{ github.ref_name }}
-        client-payload: '{"sha": "${{ github.sha }}"}'
```

Comment:
@kmruehl, don't delete this file

Reply:
resolved with https://github.com/WEC-Sim/WEC-Sim/pull/803/commits/f9946c228e1e0dc14371396663e6189992082423